templates/flat-template.svg

```diff
@@ -16,7 +16,7 @@
 
   <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11">
     {{?it.logo}}
-      <image x="5" y="3" width="{{=it.logoWidth}}" height="14" xlink:href="{{=it.logo}}"/>
+      <image x="5" y="2" width="{{=it.logoWidth}}" height="14" xlink:href="{{=it.logo}}"/>
```

Comment:
Could you include snapshots of how it looks on varying platforms to justify the change?